code/modules/reagents/chemistry/reagents/other_reagents.dm

```diff
@@ -1382,14 +1382,22 @@
 	var/current_size = 1
 
 /datum/reagent/growthserum/on_mob_life(mob/living/carbon/H)
-	if(volume >= 20 && current_size != 2)
-		H.resize = 2/current_size
-		current_size = 2
-		H.update_transform()
-	else if (current_size != 1.5)
-		H.resize = 1.5/current_size
-		current_size = 1.5
-		H.update_transform()
+	var/newsize = current_size
+	switch(volume)
+	if(0 to 19)
```

Comment:
1 more tab, yes.
(and the ```newsize = X``` lines need to be 1 tab under the ifs, so if you're moving the ifs you'll need to move those lines too)